src/quoting-client.js

```diff
@@ -35,6 +36,7 @@ class QuotingClient {
       let query = {
         source_ledger: source,
         destination_ledger: destination,
+        destination_account: destinationAccount,
```

Comment:
I don't understand the question; the connector provides quotes (by whatever algorithm it choses), not the ledger.